### ISSUE TITLE
submitStandbyRefresh: Error

### ISSUE THREAD

**Issue by dilox75** [Shardeum/shardeum-bug-reporting#675]
##### What is the issue?

After two days with a standby status, I have this message :

Node exited with the following message:
submitStandbyRefresh: Error posting standbyRefresh request: RequestError: connect ECONNREFUSED 43.134.33.175:9001

Now I already restarted it but I think I lost my place in the waiting queue. And I can see that 1001 nodes are confirmed, but you are looking for 1000 nodes max, so maybe it's too late for the other ones :/

Please help me by answering

**Comment by vinothselvaraj** [comment]
i restared it, after a day again it stopped with same error

**Comment by dilox75** [comment]
> i restared it, after a day again it stopped with same error

Yes I restarted one time also and 1 day later I have the same problem.

Where does it from? Someone can help us please?

We fall so alone :(
